exercises/string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,8 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  var string = '';
+  for (var i = word.length; i > 0; i--) {
+    string = string + word[i - 1];
```

Comment:
You can make this just use `i` if you start the loop at `length - 1` rather than the `length`. You would also need to change `> 0` to `>=`